<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="field"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Настройки</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Количество элементов до победы</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="needToWinBox">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>19</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Сохранить и начать Новую игру</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Размер поля</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sizeBox">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>19</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="startNewGame">
   <property name="text">
    <string>Начать новую игру</string>
   </property>
  </action>
  <action name="resetScore">
   <property name="text">
    <string>Сбросить счет</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>Об игре</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
